cron.sh

```diff
@@ -16,11 +16,11 @@ if [ "" == "$PKG_OK" ]; then
   fi
 fi
 
-#Check if node is installed 
-PKG_OK=$(dpkg-query -W --showformat='${Status}\n' nodejs|grep "install ok installed")
-echo "Checking for node: $PKG_OK"
-if [ "" == "$PKG_OK" ]; then
-  echo "Node not installed. Do you want to set it up? (y/n)"
+#Locate install path of node
+nodel=$(which nodejs || which node)
+echo "Checking node path: $nodel"
+if [ "$nodel" == "" ];then 
```

Comment:
This is wrong.

When the program X is not found, which returns: `X not found`
and in your case it'll output two lines when both are not found.

Reply:
```Checking for cron: install ok installed
Checking node path: 
Node not installed. Do you want to set it up? (y/n)```
In case none of the programs is found this is the output as tested. I cannot reproduce your two lines case.

Reply:
@raydwaipayan I meant, "which" command returns two lines of text and it is not empty so your check will fail.

_Sent from my Lenovo Z2 Plus using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

Reply:
@thealphadollar Changed package names to check your issue. Script successfully detects no such package installed. I do not see any issue being caused by it. Can you review it on your machine?

Reply:
![screenshot from 2018-12-20 00-05-57](https://user-images.githubusercontent.com/32812320/50240655-5339b700-03eb-11e9-8b8b-89fbb4725e9c.png)

This is what I get, and not an empty response that you are checking for.